com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -96,6 +101,27 @@ public RpcCommunicator(CommunicatorInitParameters communicatorInitParameters)
             return true;
         }
 
+        internal static bool CheckPythonPackageVersionIsCompatible(string pythonLibraryVersion)
+        {
+            Version pythonVersion;
+            try
+            {
+                pythonVersion = new Version(pythonLibraryVersion);
+            }
+            catch
```

Comment:
I don't love not specifying the exception type here, but there are a bunch of exception type that parsing the version can throw, and we'd handle all of them the same way anyway.